### ISSUE TITLE
Add version support to published docs

### ISSUE THREAD

**Issue by Vye** [tealdeer-rs/tealdeer#358]
Hi,

Can versioning be added to the docs? I'm a new user and went through a fair amount of unnecessary steps to figure out custom page extension listed in docs is for an unreleased commit in main. :(

Extension changed: https://github.com/dbrgn/tealdeer/issues/320

No `.md` extension docs for 1.6.1 tag: https://github.com/dbrgn/tealdeer/blob/v1.6.1/docs/src/usage_custom_pages.md

With `.md` extension live docs with new extension: https://dbrgn.github.io/tealdeer/usage_custom_pages.html

Thanks for the client!

**Comment by niklasmohrin** [comment]
That sounds like a good idea. If someone can come up with a not too complicated setup for this, it would be greatly appreciated :)

In the mean time, maybe we should only update github pages when we push a new release, not for every commit on main @dbrgn ?

**Comment by dbrgn** [comment]
Yeah, it would probably make sense to have versioned docs. I don't know what would be required for that though.

**Comment by dbrgn** [comment]
> In the mean time, maybe we should only update github pages when we push a new release, not for every commit on main @dbrgn ?

That might be a good stop-gap solution until then.

**Comment by niklasmohrin** [comment]
New, versioned docs are live at https://docs.tealdeer.org, old docs automatically redirect there

**Comment by niklasmohrin** [comment]
(for future reference, the site is powered by readthedocs.org)
